### PR TITLE
Update API to meta-operator

### PR DESCRIPTION
**Pull request by horis233**

### CODE REVIEW

File: deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ metadata:
     createdAt: "2020-01-06T10:16:16Z"
     description: The Common Service Operator provides a Kubernetes CRD-Based API to
       manage the lifecycle of IBM cloud common services.
-    repository: https://github.com/IBM/common-service-operator
+    repository: https://github.com/IBM/meta-operator
     support: IBM
   name: common-service-operator.v0.0.3
```
Comment (gyliu513) [inline deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml]: meta-operator?

Comment (gyliu513) [reply deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml]: How about change the file name to meta-operator as well?

Comment (chenzhiwei) [reply deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml]: let's refactor one by one.

we need to create new repository and application on quay.io for this new name.

**Pull request merged**